Gemfile

```diff
@@ -40,10 +42,7 @@ group :test do
   gem 'launchy'
 end
 
-group :production do
-  gem 'rails_12factor'
-end
-
-group :staging do
+group :production, :staging do
   gem 'rails_12factor'
+  gem 'carrierwave-aws'
```

Comment:
Since you required `carrierwave-aws` on line 20, requiring it here is duplicate.